src/form/types.ts

```diff
@@ -39,6 +40,7 @@ export interface FormLabelProps
 export interface BaseFormElementProps {
     label?: FormLabelProps | string | undefined;
     errorMessage?: string | undefined;
+    customMessage?: string | undefined;
```

Comment:
Actually in an example use case Karling showed, there is a chance of an icon in front of the message. May i suggest we deal with this in a separate ticket as an enhancement. This warrants a proper kick off to understand the requirements fully.